src/presentation/pages/Home.vue

```diff
@@ -15,11 +32,16 @@
 import { useHead } from 'unhead';
 import { useI18n } from 'vue-i18n';
 import { useAppState } from '@/application/services/useAppState';
-import NoteList from '@/presentation/components/note-list/NoteList.vue';
-
+import { CardVertical } from 'codex-ui/vue';
+import useNoteList from '@/application/services/useNoteList';
+import { getTitle, getUpdateTime } from '@/application/services/useNoteUtils';
```

Comment:
lets move these utils to `/infrastucture/utils/note`, since they are not an application service